_posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Pulp Container 2.0.0 is Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# Pulp Ansible 0.2.0 Release Announcement
+
+The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugin.
+
+## About the Pulp Ansible content plugin
+
+If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
+
+With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
+
+* Mirror a subset of Ansible content on-premise
+* Mirror all of Ansible Galaxy’s content on-premise
+* Store private Ansible content on-premise
+* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
+* Version Ansible content over time and rollback if necessary
+* Support for the new multi-role content type from Galaxy
```

Comment:
There is not an action in this. May be it should be "as well as the new multi-role type" in the sentence above.

Reply:
I will split these into workflows and features maybe?